src/Response.php

```diff
@@ -74,7 +74,7 @@ public function setOptions(array $options)
                 continue;
             }
 
-            if ($key == 'jsonrpc') {
+            if ('jsonrpc' == $key) {
```

Comment:
This doesn't do what you think it does, a fact which is made clear when you add a test with the following:

```php
$response->setOptions([0 => '2.0']);
Assert::assertNull($response->getVersion();
```

The above is identical to the previous code; it just swaps the sides of the comparison. What is required is to switch to the strict equality operator, `===`.

I've added a test, and updated the patch.